.gitignore

```diff
@@ -38,5 +38,4 @@ build/
 ### temp files ###
 *.class
 /tmp-*
-logs/
```

Comment:
let's actually keep excluding this. I have `logback` set to still dump some items into a log file for redundancy

https://github.com/softwareconstruction240/autograder/blob/339f290fb39f0afd92d7e621c80c510fb8841d35/src/main/resources/logback.xml#L22